test/transforms.ts

```diff
@@ -38,6 +38,11 @@ test('single on insert', async () => {
   await postgrest.from('users').delete().eq('username', 'foo')
 })
 
+test('maybeSingle', async () => {
+  const res = await postgrest.from('users').select().limit(0).maybeSingle()
```

Comment:
Oh, this test doesn't look right. Doing `/table?limit=0` is invalid in postgrest and will reply with a 416 Range Not Satisfiable: https://github.com/PostgREST/postgrest/issues/1121#issuecomment-392578452. I think this restriction can be lifted, but for now I'd suggest doing a test based on a nonexistent id `.eq('id', 123456)`.

Reply:
Perfect - I was in the middle of the livestream and figured I'd need your help to figure this one out 😂